src/test/java/datawave/query/util/TypeMetadataTest.java

```diff
@@ -0,0 +1,67 @@
+package datawave.query.util;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TypeMetadataTest {
+    
+    private TypeMetadata typeMetadata;
+    
+    @Before
+    public void setup() {
+        typeMetadata = new TypeMetadata();
+        typeMetadata.put("field1", "ingest1", "LcType");
```

Comment:
This is more preference than prescription, I would prefer to see field names follow the 'all-capital-letters' convention, and the datatypes be named `datatypeA`, `datatypeB`, etc. 

Feel free to ignore, this test looks pretty good.

Reply:
Simple enough - I'll change them for consistency sake.